t/mojo/daemon.t

```diff
@@ -161,6 +169,13 @@ $app->routes->any(
   }
 );
 
+my $whatever = 'Failed!';
+
+$app->hook(worker_starting => sub {
+  my ($app) = shift;
+  $whatever = 'Whatever!';
```

Comment:
Maybe test that the $app is actually the first argument?

Reply:
Yes, will add a check for that.